PracticeOop2.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+trait Active 
+{
+    public function defindYourSelf() 
+    {
+        return get_class();
+    }
+}
+
+abstract class Country
+{
+    protected $slogan;
+    
+    public function setSlogan($slogan) 
+    {
+        return $this->slogan = $slogan;
+    }
+    
+    public function getSlogan () 
+    {
+        return $this->slogan;
+	}
+
+    public function sayHello(){
+
+    }
+}
+
+interface Boss
+{
+    public function checkValidSlogan();
+}
+
+class EnglandCountry extends Country implements Boss
+{
+    use Active;
+    
+    public function sayHello()
+    {
+        return "Hello";
+    }
+    
+    public function checkValidSlogan() 
+    {
+        $str = strtolower($this->slogan);
```

Comment:
Bên trên có khai báo function getSlogan rồi , Sao dưới còn dùng $this->slogan ???